server/utils/url-utils.js

```diff
@@ -0,0 +1,37 @@
+const isDev = process.env.NODE_ENV !== 'production';
+const isBeta = !!process.env.BETA;
+
+export function getEmailSender() {
+  return process.env.EMAIL_SENDER || 'team@freecodecamp.com';
+}
+
+export function getPort() {
+  if (!isDev) {
+    return '443';
```

Comment:
Ah, cool.. I'll update, I used this from the previous code for the verify email send logic, where I think the port was used for production. But NVM, I'll update asap.